concrete/src/User/UserList.php

```diff
@@ -331,8 +331,11 @@ public function filterByGroup($group = '', $inGroup = true)
         $this->checkGroupJoin();
```

Comment:
Im pretty sure we can remove this but I dont know if anyone uses the getQueryObject after filterByGroup expecting us to have g.*/ug.* left over